src/app/shared-services/informed-consent/default.ts

```diff
@@ -0,0 +1,7 @@
+export class InformedConsent {
+  defaultInformedConsent = false
+}
+
+export class FirstTimeConsent {
+  defaultFirstTimeConsent = true
+}
```

Comment:
You could rename this file to something more expressive, e.g. `informed-constent.fixtures.ts` / `informed-constent.default.ts`.

Other than that I'd maybe combine this into one `class` or `const` instead of two `classes`:
`export const InformedConsentDefaults = {
  hasInformedConsent: false,
  hasFirstTimeConsent: true,
}
`